### DISCUSSION TITLE
Is it possible to override the code syntax highlighting language for a chunk?

### DISCUSSION THREAD

**Discussion by joelostblom** [quarto-dev/quarto-cli#10497] [Q&A]
### Description

Sometimes I use `rpy2` to include R code in a jupyter kernel via the `%%R` magic. Ideally I would like the code in these chunks to be highlighted as R code rather than Python code, is this possible?

The chunks look like this:

````
```{python}
%%R

# Some R code here
```
````

**Comment by mcanouil** [comment]
Possibly via a Lua filter that modifies the "echoed cell".

**Reply by cscheid** [reply]
That's probably the best path. @joelostblom you can add `keep-md: true` to your document to see the intermediate markdown that is provided to Quarto, and go from there.

**Reply by joelostblom** [reply]
Thank you both!
